gravitee-apim-rest-api/gravitee-apim-rest-api-standalone/gravitee-apim-rest-api-standalone-bootstrap/src/main/java/io/gravitee/rest/api/standalone/boostrap/Bootstrap.java

```diff
@@ -160,6 +160,18 @@ public static void main(String[] args) {
         mainThread.setName("graviteeio-node");
 
         try {
+            for (String arg : args) {
+                String[] argument = arg.split("=");
+                if (argument.length != 2) {
+                    throw new RuntimeException(String.format("Wrong argument was passed %s", arg));
+                } else {
+                    if (argument[0].startsWith("--")) {
+                        argument[0] = argument[0].substring(2);
+                    }
+                    System.getProperties().put(argument[0], argument[1]);
+                }
+            }
```

Comment:
what about "-D" options ? If you execute `./gravitee -Dupgrade.mode=true` you will have by default the good env var `upgrade.mode`, but also the system property `-Dupgrade.mode`